src/DataMapper/Product/Provider/ProductPrices.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\DataMapper\Product\Provider;
+
+final class ProductPrices
+{
+    public function __construct(
+        public readonly int $price,
+        public readonly int $originalPrice,
```

Comment:
I guess the original price should be nullable. Maybe I was a bit confusing on my first comment :)

Reply:
Yup, I know, I started a meeting before I had fixed that :D